### PR TITLE
Actions browserstack

### PR DESCRIPTION
**Pull request by JordanWestern**

### CODE REVIEW

File: Plugins/Specflow.Actions.Browserstack/Specflow.Actions.Browserstack/BrowserstackDriverInitialiser.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        {
+            var options = new ChromeOptions();
+
+            options.AddAdditionalCapability("browserstack.user", BrowserstackUsername.Value, true);
+            options.AddAdditionalCapability("browserstack.key", AccessKey.Value, true);
+            options.AddAdditionalCapability("name", GetScenarioTitle(), true);
+
+            if (capabilities?.Count != 0 && capabilities != null)
+            {
+                foreach (var capability1 in capabilities)
```
Comment (SabotageAndi) [inline Plugins/Specflow.Actions.Browserstack/Specflow.Actions.Browserstack/BrowserstackDriverInitialiser.cs:39]: I think we can rename this to `capability`.

File: Plugins/Specflow.Actions.Browserstack/Specflow.Actions.Browserstack/BrowserstackDriverInitialiser.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                {
+                    options.AddAdditionalCapability(capability1.Key, capability1.Value, true);
+                }
+            }
+
+            var capability = options;
```
Comment (SabotageAndi) [inline Plugins/Specflow.Actions.Browserstack/Specflow.Actions.Browserstack/BrowserstackDriverInitialiser.cs:45]: We can remove this line.

**Review by SabotageAndi — CHANGES_REQUESTED** [review]
Very good PR! I found two minor minor possibilities for improvements.

File: Plugins/Specflow.Actions.Browserstack/Specflow.Actions.Browserstack/BrowserstackDriverInitialiser.cs (hunk 1 above)
Comment (JordanWestern) [reply Plugins/Specflow.Actions.Browserstack/Specflow.Actions.Browserstack/BrowserstackDriverInitialiser.cs:39]: Didn't notice this!

**Review by SabotageAndi — APPROVED** [review]

**Pull request merged**